compiler/env/ProcessorInfo.hpp

```diff
@@ -46,9 +46,9 @@ enum TR_S390MachineType
    TR_ZEC12_RESERVE           =  2829,  // reserve for zHelix
    TR_Z13                     =  2964,  // z13
    TR_Z13s                    =  2965,  // z13s
+   TR_ZNEXT                   =  3906,  // zNext
+   TR_ZNEXTs                  =  3907,  // zNexts
```

Comment:
Just for my information, what is the difference between TR_ZNEXT and TR_ZNEXTs in this context ?